edx_proctoring/migrations/0007_auto_20180220_1409.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
This isn't a blocker if it's too much trouble, but could you attempt to re-create these migrations as a single file, with a descriptive name?

How-to:
1. unapply migrations locally (migrate `edx_proctoring` back to 0005)
2. delete these 2 migration files
3. re-run `makemigrations`, but use the `-n NAME` flag to give the operations a useful name (https://docs.djangoproject.com/en/1.11/ref/django-admin/#django-admin-makemigrations)